### PR TITLE
Create Phone field component

### PR DESCRIPTION
**Pull request by sdk-irl**
## Description

Use the web components to create a custom Telephone field that will have built in validation.

## Original issue(s)
department-of-veterans-affairs/va-forms-system-core#210

## Testing done
Tested for the correct attributes, correct and incorrect responses, and error messages
## Screenshots
Screen Shot 2022-05-09 at 3 27 15 PM

## Definition of done
- [ ] Documentation has been updated, if applicable
- [x] A link to the original github issue has been provided
- [x] No sensitive information (i.e. PII/credentials/internal URLs etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by micahchiang — APPROVED** [review]
Looks great @sdk-irl! Great work.

File: examples/index.jsx [hunk 1]
```diff
@@ -14,7 +14,8 @@ const initialValues = {
   streetTwo: '', 
   streetThree: '', 
   state: '', 
-  zipcode: ''
+  zipcode: '',
+  phone: ""
```
Comment (bjmccotter7192) [inline examples/index.jsx:18]: Can you change this to single quotes to match the others?

**Review by bjmccotter7192 — COMMENTED** [review]
LGTM! Great work.

File: src/form-builder/PhoneField.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  return (
+    <TextField
+      id={id}
+      {...props}
+      onChange={field.onChange}
+      value={value}
+      required
+      onBlur={() => helpers.setTouched(true)}
+    />
```
[the remark this answers is not in the document]
Comment (sdk-irl) [reply src/form-builder/PhoneField.tsx]: Thanks @bkjohnson, added this in and relayed to the team to make sure we get them in the other appropriate components as well.

**Pull request merged**